<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-S6POP2D" ignored="0" name="Test" tests="3" failures="2" timestamp="06 Jun 2022 02:33:00 GMT" time="2.317" errors="0">
  <testcase name="logintest" time="1.783" classname="com.Test.LoginTest"/>
  <testcase name="failTest" time="0.01" classname="com.Test.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [test] but found [JavaByKiran | Dashboard]">
      <![CDATA[java.lang.AssertionError: expected [test] but found [JavaByKiran | Dashboard]at com.Test.LoginTest.failTest(LoginTest.java:40)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- failTest -->
  <testcase name="failTest" time="0.371" classname="com.Test.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat java.io.File.<init>(Unknown Source)at com.utility.DriverUtils.getScreenshot(DriverUtils.java:19)at com.listner.MyListner.onTestFailure(MyListner.java:29)... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- failTest -->
</testsuite> <!-- Test -->
